apiary.apib

```diff
@@ -2633,6 +2633,8 @@ The create request is created and added to a queue, the job resource url for sta
 
 ### Create new table from time travel asynchronously [POST]
 
+Supported by Snowflake and Bigquery, Bigquery doesn't support primary keys so if you create a table using time travel for Bigquery backend the new table won't get primary keys.
+
```

Comment:
Bylo by fajn kdyby na to upozrňovalo UI taky asi udělat task

Reply:
https://keboola.atlassian.net/browse/BIG-116